response.go

```diff
@@ -93,6 +93,14 @@ type CommonV3Response struct {
 	Time       int         `json:"time"`
 }
 
+type V5Response[T any] struct {
+	RetCode    int         `json:"retCode"`
+	RetMsg     string      `json:"retMsg"`
+	RetExtInfo interface{} `json:"retExtInfo"`
+	Time       int         `json:"time"`
+	Result     T           `json:"result"`
+}
```

Comment:
It looks like a good change, but CommonV5Response is used here and we do not want the definitions to be scattered.

https://github.com/hirokisan/bybit/blob/35957be13d796ff2323727fa0f4f2b5353d13e53/response.go#L124-L126